internal/cli/atlas/alerts/list_test.go

```diff
@@ -23,24 +23,25 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
 	mocks "github.com/mongodb/mongodb-atlas-cli/internal/mocks/atlas"
+	"github.com/mongodb/mongodb-atlas-cli/internal/pointer"
 	"github.com/stretchr/testify/assert"
-	"go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas-sdk/admin"
 )
 
 func TestList_Run(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockStore := mocks.NewMockAlertLister(ctrl)
 
-	expected := &mongodbatlas.AlertsResponse{
+	expected := &admin.PaginatedAlert{
 		Links: nil,
-		Results: []mongodbatlas.Alert{
+		Results: []admin.AlertViewForNdsGroup{
 			{
-				ID:            "test",
-				EventTypeName: "test",
-				Status:        "test",
-				MetricName:    "test"},
+				Id:            pointer.Get("test"),
```

Comment:
[Unrelated] We need to document SDK methods that create integer or string pointers so users can use SDK without having to rely on pointer lib etc.

Reply:
creating jira